projects/m2/014-sublists/js/index.js

```diff
@@ -0,0 +1,23 @@
+const prompt = require("prompt-sync")();
+
+
+function isSublist(){
+    const list = "25,30,35,40".split(",");
+    const sublist = "25,30,35".split(",");
+    const indexControl = [];
+
+    for(i=0;i<sublist.length;i++){
+        indexControl.push(list.indexOf(sublist[i]))
+    }
+    for(j=0;j<indexControl.length;j++){
+        if(indexControl[j]+1 === indexControl[j+1] || sublist.join("") === ""){
+            return true;
+        }else{
+            return false;
+        }
+    }
+
+    console.log(indexControl);
+}   
+
+console.log(isSublist());
```

Comment:
The exercise is not performed according to the requirements. 
Check why the function with sublist **[10,20,40]** returns `true` instead of `false`.